### PR TITLE
Added release notes script (connect #604)

### PR DESCRIPTION
**Pull request by valllllll2000**
#### Before the PR (what is the issue or what needed to be done)
Release notes had to be generated manually copy-pasting issues and their url
#### The solution
You can pre-generate a list of issues with the script and then add the description by hand
#### Screenshots (if appropriate)
NA
#### Reviewer Checklist
* [ ] Connect the issue
* [ ] Test plan
* [ ] Copyright header
* [ ] Code formatting
* [ ] Documentation

### CODE REVIEW

File: util/scripts/release_notes.py [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+# usage : update milestone to one you want to generate release notes for
+# run with python release_notes.py
+# Your notes will be written to the file temp.md
+
+import json
+import time
+import urllib2
+
+milestone = '2.2.11'
+label_new = '%22New%20and%20noteworthy%22'
```
Comment (nhternup) [inline util/scripts/release_notes.py]: Could be a list
```
labels = ['New and noteworthy', 'Resolved issues']
```

File: util/scripts/release_notes.py [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+label_resolved = '%22Resolved%20issues%22'
+repo = 'akvo/akvo-flow-mobile'
+url_new = 'https://api.github.com/search/issues?q=label:%22Ready%20for%20release%22+label:' + label_new + '+milestone:' + milestone + '+repo:' + repo
+url_resolved = 'https://api.github.com/search/issues?q=label:%22Ready%20for%20release%22+label:' + label_resolved + '+milestone:' + milestone + '+repo:' + repo
```
Comment (nhternup) [inline util/scripts/release_notes.py]: maybe merge into
```
api_url = 'https://api.github.com/search/issues?q=milestone:' + milestone + '+repo:' + repo + '+label:'
```

File: util/scripts/release_notes.py [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+
+    Returns
+    -------
+    json array
+        containing issues
+
+    """
+    github_request = urllib2.urlopen(url)
+    if not github_request:
+        parser.error('Error getting issues list.')
```
Comment (nhternup) [inline util/scripts/release_notes.py]: 'parser' not defined. needs...
```
import argparse
parser = argparse.ArgumentParser()
```

File: util/scripts/release_notes.py [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+
+    Returns
+    -------
+        nothing
+    """
+    for issue in issues:
+        f.write('* **' + issue['title'] + '** - [#' + str(issue['number']) + '] (' + issue[
+            'html_url'] + ')\n')
+
+f = open('temp.md', 'w')
```
Comment (nhternup) [inline util/scripts/release_notes.py]: could be global variable:
```
output_filename = 'temp.md'
```

File: util/scripts/release_notes.py [hunk 5]
```diff
@@ -0,0 +1,76 @@
...
+f.write('\n# New and noteworthy\n')
+issues = load_issues(url_new)
+write_issues(f, issues)
+
+f.write('\n')
+f.write('\n# Resolved issues\n')
+issues = load_issues(url_resolved)
+write_issues(f, issues)
+
+f.closed
```
Comment (nhternup) [inline util/scripts/release_notes.py]:
```
f.close()
```

File: util/scripts/release_notes.py [hunk 6]
```diff
@@ -0,0 +1,76 @@
...
+        nothing
+    """
+    for issue in issues:
+        f.write('* **' + issue['title'] + '** - [#' + str(issue['number']) + '] (' + issue[
+            'html_url'] + ')\n')
+
+f = open('temp.md', 'w')
+f.write('# ver ' + milestone + '\n')
+f.write('Date: '+ time.strftime("%d %B %Y"))
+f.write('\n')
```
Comment (nhternup) [inline util/scripts/release_notes.py]: If using list as in 'labels'. This section could be
```
for label in labels:
    f.write('\n\n# ' + label + '\n')
    write_issues(f, load_issues(api_url + '"' + label.replace(" ", "+") + '"'))
```

**Comment by nhternup** [comment]
@valllllll2000 Good idea on this release notes generation. Nice time saver

**Comment by valllllll2000** [comment]
@nhternup Thank you for taking the time to review my script which was very ugly :), all fixed now!

**Pull request merged**